### PR TITLE
support both python2 and python3 via six.moves

### PR DESCRIPTION
**Pull request by jguffey**
As on tin,

I found that this project did not work locally because I am running Python 2.6, however in prod I run Python 3, so I thought I'd add support for both via `six`.

### CODE REVIEW

File: shared_session/templatetags/shared_session.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import copy
 import json
-from urllib.parse import urljoin
+from six.moves.urllib.parse import urljoin
```
Comment (ViktorStiskala) [inline shared_session/templatetags/shared_session.py:3]: This requires `six` as a dependency in `setup.py` → `install_requires`.

**Review by ViktorStiskala — CHANGES_REQUESTED** [review]
Hello Josh, thanks for the contribution. I would, however, recommend upgrading Python on your local machine, py3 has so much more to offer. It should be a pretty easy and straightforward process as there are many options how to install Python nowadays.

In order for this change to work correctly, could you please make the changes also to setup.py (`install_requires` and `classifiers`)?

**Comment by jguffey** [comment]
hi @ViktorStiskala, thank you for the feedback. 

I'd love to support only 3, but I do require support for version 2 for now, as my migration to 3 is yet to be completed.

I think that I have added the correct requirements to `setup.py` but please do let me know if I'm missing anything else. I'm new to Python packages so it's possible.

Thanks!

**Comment by ViktorStiskala** [comment]
Thanks for the contribution.

I've uploaded the current version to pypi, so you can now install it directly using `pip install --upgrade django-shared-session`.

**Pull request merged**